src/main.js

```diff
@@ -157,9 +157,13 @@ import { VideoController } from "./components/video-controller";
     }
 
     function renderPlaybackPage() {
-        videoController.startVideo(currentVideoStream);
+        const showControlBar = true;
+        videoController.startVideoLater(currentVideoStream, showControlBar);
```

Comment:
it has to add the video to the page, since that does not exist yet at this point

the idea is that the video is always removed if playback is not occurring, i.e. when we show other pages besides playback, or when we are showing an ad on the playback page.